src/App.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { useState } from "react";
+import './App.css';
+import { Counter } from './Components/Counter/Counter';
+import { GenreSelect } from './Components/GenreSelect/GenreSelect';
+import { SearchForm } from './Components/SearchForm/SearchForm';
+
+function App() {
+  const [selectedGenre, setSelectedGenre] = useState('All');
+  const genres = ['All', 'Comedy', 'Horror', 'Crime'];
```

Comment:
Why did you decide to store genres state on app level instead of inside `GenreSelect` component?

Reply:
since the task states that this component should take in props "name of currently selected genre" and onSelect callback which takes in the selected genre name, I decided that it's a hint to manage this state outside of the component